project-base/docker/conf/docker-compose-mac.yml.dist

```diff
@@ -97,15 +98,8 @@ services:
             - discovery.type=single-node
 
     microservice-product-search:
-        build:
-            context: ../microservices/product-search/docker
-            args:
-                www_data_uid: 501
-                www_data_gid: 20
+        image: shopsys/shopsys:microservice-test
         container_name: shopsys-framework-microservice-product-search
-        volumes:
-            - shopsys-framework-microservice-product-search-sync:/var/www/html
-            - ../microservices/product-search/docker/php-fpm/php-ini-overrides.ini:/usr/local/etc/php/php.ini
```

Comment:
So right now, we do not use our php-ini for project-base?

Reply:
I have answered this in the previous comment.